charts/monitoring/prometheus/rules/src/general/blackbox-exporter.yaml

```diff
@@ -26,37 +26,3 @@ groups:
           resource: "{{ $labels.instance }}"
           service: blackbox-exporter
 
-      - alert: HttpProbeSlow
-        annotations:
-          message: "{{ $labels.instance }} takes {{ $value }} seconds to respond."
-          runbook_url: https://docs.kubermatic.com/kubermatic/main/cheat-sheets/alerting-runbook/#alert-httpprobeslow
-        expr: sum by (instance) (probe_http_duration_seconds) > 3
-        for: 15m
-        labels:
-          severity: warning
-          resource: "{{ $labels.instance }}"
-          service: blackbox-exporter
-        runbook:
-          steps:
-            - Check the target system's resource usage for anomalias.
-            - Check if the target application has been recently rescheduled and is still settling.
-
-      - alert: HttpCertExpiresSoon
```

Comment:
This alert seems very useful IMO, do we have a replacement for it?

Reply:
Didnt find any replacement in Mixin repo so re-added those alerts in the latest commit.

Reply:
I think all the blackbox alerts should stay since we ship blackbox by default with KKP. Were there any problems with those in live systems?

Reply:
Retained those alerts now